packages/wonder-blocks-tooltip/src/components/tooltip.tsx

```diff
@@ -258,6 +265,9 @@ export default class Tooltip extends React.Component<Props, State> {
 
         const popperHost = this._getHost();
 
+        const shouldBeVisible =
+            popperHost && (active || activeBubble) && this.state.anchorElement;
```

Comment:
**question:** Why do we need to now check the existence of the anchor element when we didn't before?

Reply:
This is so the `MutationObserver` can observe the correct/existent anchor element in the DOM. I'll add a comment about that.